src/topnav/topnav.component.tsx

```diff
@@ -22,6 +15,20 @@ export default function Topnav(props: TopnavProps) {
           alt="OpenMRS Logo"
         />
       </a>
+      <ul>
+        <li>
+          <NavLink to="/patient-search" activeClassName={styles.activeLink}>
+            <span>Patient Search</span>
+          </NavLink>
+        </li>
+      </ul>
+      <ul>
+        <li>
+          <NavLink to="/log-out" activeClassName={styles.activeLink}>
```

Comment:
To be consistent with how the `/login` route is spelled, I'd prefer this route to be `/logout` (without the `-`)

Additionally, we can just do the following instead of a NavLink and it will redirect to refapp instead of staying within spa:

```jsx
<a href="/logout">
  <span>Logout</span>
</a>
```